packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -22,8 +22,6 @@ import * as stylesImport from './DatePicker.scss';
 const styles: any = stylesImport;
 
 export interface IDatePickerState {
-  /** The currently focused date in the drop down, but not necessarily selected */
-  navigatedDate?: Date;
```

Comment:
You are removing this from the interface, can this break some existing consumers of this code?